pages/your_tasks/data_brokering.md

```diff
@@ -0,0 +1,89 @@
+
+### Solutions
+Collection of metadata can be done in various ways:
+
+* Electronic forms (e.g. online form, eCRF) allow controlling data at the entry point, ensuring the use of controlled vocabularies and ontologies if properly set up. While user friendly for single uploads, this type of upload can become very cumbersome for batch upload.
+* Spreadsheet files enable controlling data at the entry point, thanks to native validation features of spreadsheet tools (e.g Excel). It is at the same time very convenient for batch uploads and users generally like having a file that they can store and easily re-open to check what they have provided.
+* Text files (TSV, CSV, JSON, XML…) are very practical to automate batch submissions on the user side, since scripts can easily generate these files. It is recommended to specify to the users which encoding should be used (e.g. UTF-8). There is however no data control at the entry point and only upon submission to the data brokering database. All data validation is therefore performed upon loading into the database, and error handling should be carefully evaluated prior to implementation. Examples include automatically informing the user of errors and asking them to re-submit, or involving data curators who may correct obvious mistakes and can be in contact with data providers to clarify data validation errors. Some automation may also be implemented to map terms to a controlled list of terms or an ontology even in the case of minor typos.
+
+## Sharing data to public repositories
+### Description
+Often, a goal of the broker is to submit the collected and harmonised data in an open archive for the benefit of both the data owners and the wider scientific community. This supports sharing data with collaborators, referencing for publication and provides a [long-term storage solution](https://rdmkit.elixir-europe.org/storage.html).
+
+### Example for genomic data
+For sharing of sequencing data within Europe, the [European Nucleotide Archive](https://www.ebi.ac.uk/ena/browser/about) can be used. The ENA is part of the [INSDC](https://www.insdc.org/), a global collaboration for the archival of nucleotide sequence data including NCBI and DDBJ. For information on the structure of ENA content, please see here: https://www.ebi.ac.uk/ena/browser/about/content. A general guide to submissions to the ENA can be found here: https://ena-docs.readthedocs.io/en/latest/submit/general-guide.html
+
+In the particular case of SARS-CoV-2 genomic data, more information will be available on the SARS-CoV-2 domain page (under construction, coming soon).
+
+### Considerations
+* There are many data repositories, so it is important to decide where you plan to share the data and adhere to the correct standards and submission protocols for your data type. This may notably include adhering to checklists provided by the chosen data repositories. See here for more information: https://rdmkit.elixir-europe.org/data_publication.html.
```

Comment:
```suggestion
* There are many data repositories, so it is important to decide where you plan to share the data and adhere to the correct standards and submission protocols for your data type. This may notably include adhering to checklists provided by the chosen data repositories. More information can be found on our [data publication](data_publication) page.
```
urls should always be descriptive for accessibility reasons :)

Reply:
Got it! Thank you so much, @bedroesb!